public/utils/contextMenu/getActions.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { i18n } from '@osd/i18n';
+import { EuiIconType } from '@elastic/eui/src/components/icon/icon';
+import { toMountPoint } from '../../../../../src/plugins/opensearch_dashboards_react/public';
+import { Action } from '../../../../../src/plugins/ui_actions/public';
+import { createADAction } from '../../action/ad_dashboard_action';
+import AnywhereParentFlyout from '../../components/FeatureAnywhereContextMenu/AnywhereParentFlyout';
```

Comment:
Because we have the `index` file in `components/FeatureAnywhereContextMenu`, you can shorten the import to just be `../../components/FeatureAnywhereContextMenu`